composer/datasets/imagenet_hparams.py

```diff
@@ -211,31 +212,65 @@ class StreamingImageNet1kHparams(DatasetHparams):
     """DatasetHparams for creating an instance of StreamingImageNet1k.
 
     Args:
+        version (int): Which version of streaming to use. Default: ``2``.
         remote (str): Remote directory (S3 or local filesystem) where dataset is stored.
-            Default: ``'s3://mosaicml-internal-dataset-imagenet1k/mds/1/```
+            Default: ``'s3://mosaicml-internal-dataset-imagenet1k/mds/2/```
         local (str): Local filesystem directory where dataset is cached during operation.
             Default: ``'/tmp/mds-cache/mds-imagenet1k/```
         split (str): The dataset split to use, either 'train' or 'val'. Default: ``'train```.
         resize_size (int, optional): The resize size to use. Use -1 to not resize. Default: ``-1``.
         crop size (int): The crop size to use. Default: ``224``.
     """
 
+    version: int = hp.optional('Version of streaming (1 or 2)', default=2)
```

Comment:
Can we update the following YAML's as well with an explicit version so it's explicit?
```
composer/yamls/models/resnet50_streaming.yaml:  streaming_imagenet1k:
composer/yamls/models/resnet50_streaming.yaml:  streaming_imagenet1k:
```

Reply:
Done